roborock/api.py

```diff
@@ -189,6 +191,8 @@ async def _async_response(self, request_id: int, protocol_id: int = 0) -> tuple[
             queue = RoborockFuture(protocol_id)
             self._waiting_queue[request_id] = queue
             (response, err) = await queue.async_get(QUEUE_TIMEOUT)
+            if response == "unknown_method":
```

Comment:
I probably got this a feel times but I cant remember if the response is just this code unknown_method. Is it?

Reply:
At least sometimes it is. That's the response I got for valley timer as example

Reply:
I'm not sure why the response is that instead of the err

Reply:
I would assume it always returns in English and never changes but I'm not 100% on that

Reply:
Valley timer does work for you?

Reply:
It does not. It's not an option for my device